scalafix-testkit/src/main/scala/scalafix/testkit/RuleTest.scala

```diff
@@ -31,36 +33,54 @@ object RuleTest {
       val input = test.toInput
       val dialect = args.scalaVersion.dialect(args.sourceScalaVersion)
       val tree = dialect(input).parse[Source].get
-      val comment = SemanticRuleSuite.findTestkitComment(tree.tokens)
-      val syntax = comment.syntax.stripPrefix("/*").stripSuffix("*/")
-      val conf = Conf.parseString(test.testName, syntax).get
-      val scalafixConfig = conf.as[ScalafixConfig].get
-      val doc = v1.SyntacticDocument(
-        tree.pos.input,
-        LazyValue.now(tree),
-        DiffDisable.empty,
-        scalafixConfig
-      )
-      val sdoc =
-        v1.SemanticDocument.fromPath(
-          doc,
-          test.semanticdbPath,
-          classLoader,
-          symtab
+      // Its possible to have comments which isn't valid HOCON (i.e.
+      // license headers), so lets parse until we find valid HOCON.
+      val allComments =
+        SemanticRuleSuite.filterPossibleTestkitComments(tree.tokens)
+      val lazilyParsedComments = allComments.view.map { comment =>
+        val syntax = comment.syntax.stripPrefix("/*").stripSuffix("*/")
+        for {
+          conf <- Try(Conf.parseString(test.testName, syntax)).toEither
+            .flatMap(_.toEither)
```

Comment:
@bjaglin I didn't realize you were working on this right now, thanks for fixing the bug!

I think there is a slight improvement that can be done here, this expression ends up getting the type `Either[Serializable, _]` because the first `toEither` has `Throwable` on the left where as the second has `ConfError`, should I fix this?

Reply:
I am planning to cut a release tonight ahead of the official release of scala 2.13.12, that's why I wanted to get this out ASAP.

As for the type, I am not sure it really matters since we never unpack the left? I'll merge this if it's green in an hour or so, feel free to push if you see further improvements.

Reply:
I already have the fix written, its quite simple. Just running scalafmt/scalafix locally and ill push immediately after